[2022-02-02 21:28:54,211] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.halo scheduled__2019-11-02T06:00:00+00:00 [queued]>
[2022-02-02 21:28:54,393] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.halo scheduled__2019-11-02T06:00:00+00:00 [queued]>
[2022-02-02 21:28:54,393] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:28:54,394] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-02 21:28:54,394] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:28:54,507] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): halo> on 2019-11-02 06:00:00+00:00
[2022-02-02 21:28:54,514] {standard_task_runner.py:52} INFO - Started process 1504 to run task
[2022-02-02 21:28:54,527] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'halo', 'scheduled__2019-11-02T06:00:00+00:00', '--job-id', '359', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmpycgfp1_o', '--error-file', '/tmp/tmpjqq2wjmc']
[2022-02-02 21:28:54,534] {standard_task_runner.py:77} INFO - Job 359: Subtask halo
[2022-02-02 21:28:54,823] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.halo scheduled__2019-11-02T06:00:00+00:00 [running]> on host d574c7c5b085
[2022-02-02 21:28:55,014] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=halo
AIRFLOW_CTX_EXECUTION_DATE=2019-11-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-11-02T06:00:00+00:00
[2022-02-02 21:28:55,046] {logging_mixin.py:109} INFO - hello world!!
[2022-02-02 21:28:55,047] {python.py:175} INFO - Done. Returned value was: None
[2022-02-02 21:28:55,156] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=halo, execution_date=20191102T060000, start_date=20220202T212854, end_date=20220202T212855
[2022-02-02 21:28:55,314] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:28:55,578] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
